MobileWallet/Backup/ICloudBackup.swift

```diff
@@ -202,6 +202,15 @@ class ICloudBackup: NSObject {
                 fileURL = try zipWalletDatabase()
             }
 
+            if TariSettings.shared.isUnitTesting() {
```

Comment:
```suggestion
            if TariSettings.shared.isUnitTesting {
```

This is probably why tests are failing